docs/using-octavia-ingress-controller.md

```diff
@@ -64,7 +65,7 @@ kubectl apply -f /etc/kubernetes/octavia-ingress-controller/serviceaccount.yaml
 
 ### Prepare octavia-ingress-controller configuration
 
-The octavia-ingress-controller needs to communicate with OpenStack cloud to create resources corresponding to the Kubernetes ingress resource, so the credentials of an OpenStack user(doesn't need to be the admin user) need to be provided. Additionally, in order to differentiate the ingresses from different kubernetes cluster, `cluster_name` needs to be unique.
+The octavia-ingress-controller needs to communicate with OpenStack cloud to create resources corresponding to the Kubernetes Ingress resource, so the credentials of an OpenStack user(doesn't need to be the admin user) need to be provided. Additionally, in order to differentiate the Ingresses between kubernetes clusters, `cluster_name` needs to be unique.
```

Comment:
since we are enhancing this doc, maybe consider reword
`so the credentials of an OpenStack user(doesn't need to be the admin user) need to be provided.`
to something or reference that point to the main section about cloud.conf to avoid confusion?